ansible/group_vars/all.yml

```diff
@@ -27,6 +27,11 @@ cluster_name: cluster.local
 insecure_registrys:
 #  - "gcr.io"
 
+# Required for CoreOS. CoreOS does not include a Python interpreter. The
+# pre-ansible role installs a python interpreter to /opt/bin/. For more
+# information see https://coreos.com/blog/managing-coreos-with-ansible.html
+#ansible_python_interpreter: "PATH=/opt/bin:$PATH python"
```

Comment:
According to the following, this parameter can only be set in 3 places and group_vars seems to me the most applicable for my use case:

http://stackoverflow.com/questions/22769568/system-specific-variables-in-ansible